config/routes.rb

```diff
@@ -1,10 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+ 
+  get '/user', to: 'user#index', as: 'users'
+  
+  get '/user/:id', to: 'user#show', as: 'user'
 
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
+  get '/posts/', to: 'post#index'
+  get '/posts/show', to: 'post#show'
```

Comment:
- [ ] Please make sure that you are following the right convention for the routes of the application as instructed in the `project` requirements. As per the requirement your routes should work in a way that, if any person, opens the `posts` then make sure that you need to only show the post of the user with a specific ID, and if a person wants to try to open a specific post then it should show the specific post of the specific `user`. Kindly look at the below suggestion to understand things very well.

 ``` suggestion
  get '/user/:id/posts/', to: 'post#index'
  get '/user/:user_id/posts/:id', to: 'post#show'
```

### OR 

#### Simply you can configure your routes as follows:

```rb
resources: users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

### Requirement

![image](https://github.com/AlaaAlsalem/blogaapp/assets/66476329/034ace9d-7016-4b73-8c96-e5a8dc880e94)